tools/SetupFlow/DevHome.SetupFlow/Services/DevDriveManager.cs

```diff
@@ -301,7 +304,7 @@ private DevDrive GetDevDriveWithDefaultInfo()
         }
 
         var devDriveState = validationSuccessful ? DevDriveState.New : DevDriveState.Invalid;
-        return new Models.DevDrive(driveLetter, DevDriveUtil.MinDevDriveSizeInBytes, ByteUnit.GB, DefaultDevDriveLocation, fileName, devDriveState, Guid.NewGuid());
+        return new Models.DevDrive(driveLetter, DevDriveUtil.MinDevDriveSizeInBytes, DevDriveUtil.MinDevDriveSizeInBytes, ByteUnit.GB, DefaultDevDriveLocation, fileName, devDriveState, Guid.NewGuid());

```

Comment:
The second DevDriveUtil.MinDevDriveSizeInBytes indicates the size remaining. Since this is invoked while creating a new dev drive, size remaining is the same as the total size. I have left a comment explaining the parameter.